hack/local-development/common/helpers

```diff
@@ -142,8 +142,15 @@ get_host_address () {
   local ip_route=""
   local ip_address=""
 
+  # TODO: SC2235
```

Comment:
Sorry?

Reply:
I forgot, the line below the TODO can be optimized by grouping instead of many subshells.
Docs: https://www.shellcheck.net/wiki/SC2235

Not really necessary you can remove the comment if you want.

Reply:
😵‍💫 Either fix it now or remove it, I doubt that anybody will pick this TODO up otherwise 😉